stock_account_quantity_history_location/readme/CONSTRIBUTORS.rst

```diff
@@ -3,3 +3,5 @@
 * `Tecnativa <https://www.tecnativa.com>`_:
 
   * Víctor Martínez
+
+* Sodexis <dev@sodexis.com>
```

Comment:
I noticed the filename has a type. Can you rename it to CONTRIBUTORS.rst please?

Reply:
Done